src/Raven.Studio/wwwroot/App/views/database/tasks/ongoingTasks.html

```diff
@@ -244,12 +244,9 @@ <h5 class="tasks-list-item no-text-transform" title="Navigate to the Server-Wide
                                         <label></label>
                                     </div>
                                     <div class="panel-name flex-grow">
-                                        <h3>
-                                            <span>
-                                                <a href="#" data-bind="click: editTask, attr: { 'title': 'Task name: ' + taskName() }">
-                                                    <span data-bind="text: taskName"></span>
-                                                </a>
-                                            </span>
+                                        <h3 data-bind="attr: { 'title': 'Task name: ' + taskName() }">
+                                            <a href="#" data-bind="if: !isServerWide(), click: editTask"><span data-bind="text: taskName"></span></a>
+                                            <a data-bind="if: isServerWide()"><span data-bind="text: taskName"></span></a>
```

Comment:
can we avoid using `a` tag when it doesn't have href?

Reply:
If I use `span `instead of `a `then text will be 'gray', not 'red',
and it will look different from the other tasks.
If that is ok - then I will change to `span`.